javascript/elements/updates_for_element.js

```diff
@@ -41,9 +41,8 @@ export default class UpdatesForElement extends SubscribingElement {
   }
 
   async update (data) {
```

Comment:
I think you can remove the `async` from the `update()` function now that you moved the async part to the `processBlock()` function

Reply:
good spot!